index.html

```diff
@@ -251,19 +251,16 @@ <h1 class="name">Start Bootstrap</h1>
                         <h3>Around the Web</h3>
                         <ul class="list-inline">
                             <li>
-                                <a href="#" class="btn-social btn-outline"><span class="sr-only">Facebook</span><i class="fa fa-fw fa-facebook"></i></a>
+                                <a href="https://github.com/CodeForPoznan" class="btn-social btn-outline"><span class="sr-only">GitHub</span><i class="fa fa-fw fa-github"></i></a>
                             </li>
                             <li>
-                                <a href="#" class="btn-social btn-outline"><span class="sr-only">Google Plus</span><i class="fa fa-fw fa-google-plus"></i></a>
+                                <a href="https://gitter.im/CodeForPoznan" class="btn-social btn-outline"><span class="sr-only">Gitter</span><i class="fa fa-fw fa-gitter"></i></a>
                             </li>
                             <li>
-                                <a href="#" class="btn-social btn-outline"><span class="sr-only">Twitter</span><i class="fa fa-fw fa-twitter"></i></a>
+                                <a href="https://www.linkedin.com/company-beta/10955733/" class="btn-social btn-outline"><span class="sr-only">Linked In</span><i class="fa fa-fw fa-linkedin"></i></a>
```

Comment:
<span class="sr-only">Linked In</span> -> LinkedIn. Official naming is without space.